### ISSUE TITLE
Lack of good UX

### ISSUE THREAD

**Issue by patrick-elmquist-knowit** [BoostIO/Boostnote#2243]
## TL;DR
I think the project lack UX vision. Some developers creating PRs just add what they feel like in the moment with no thought to the visual appearance or UX. It makes it harder to find and use features and settings in the app and I think that PRs should not just be approved based on the quality of the code but the visuals as well.

## Long version

I think Boostnote started out great when there's was just a few features, but as its been growing without any proper UX vision it's starting to feel really messy. It seems it's always the simplest solution that is chosen, no matter if it makes sense from a user perspective or not.

## Examples:
The Settings window is a mess, none of the pages look the same. There are different spacings and alignments both between and within pages.

### Alignment
* The Interface page is particularly bad, where nothing is aligned and the sub categories are cramped together into one big ball of mess.

![screen shot 2018-07-24 at 21 12 12](https://user-images.githubusercontent.com/36044123/43160721-45c8e142-8f86-11e8-86b5-6875bd33a2e9.png)

### Common Visual Design
* Why are there so many different types of buttons/actions? All of these and more are found in Settings
![untitled-1](https://user-images.githubusercontent.com/36044123/43161352-2b24b7b0-8f88-11e8-804a-300ce1b934ab.png)

* Who thought this was a job well done? It looks so half assed that I get a tiny bit angry every time I see it.
![screen shot 2018-07-24 at 21 11 54](https://user-images.githubusercontent.com/36044123/43160734-525d84ee-8f86-11e8-95e5-0a3054d2634a.png)

### Other user experience findings
* Auto closing `*` is a really bad idea considering it's also used for creating lists. It's really annoying that every time you create a new list you get `* *`
* No feature descriptions, for example what the heck is a Snippet and how do you use it? Well unless I go check the Change Log or the closed PRs I have no idea.
* The About and Crowdfounding pages are for some reason positioned in the middle of pages containing actual settings

**Comment by modmod24** [comment]
don't forget to mention #2237. there had been many enhancements with this release, a pre release for testing would have been nice.

**Comment by Rokt33r** [comment]
@patrick-iv-knowit Could you join our slack? We could discuss more in there.

**Comment by Flexo013** [comment]
@patrick-elmquist-knowit Do you still wish to help figure out with improving the UX of Boostnote?

**Comment by Flexo013** [comment]
I'm closing this issue do to prolonged inactivity.

If you still wish to help with improving the UX then feel free to reopen this issue.
